### PR TITLE
Document the Snaps static file API

### PR DESCRIPTION
**Pull request by ziad-saab**
Fixes MetaMask/metamask-planning#1491

### CODE REVIEW

**Comment by ziad-saab** [comment]
**TODO**: Add feature icon

File: snaps/index.mdx [hunk 1]
```diff
@@ -83,6 +83,12 @@ The following Snaps features are available in the stable version of MetaMask:
       href: "reference/permissions#endowmentlifecycle-hooks",
       title: "Lifecycle hooks",
       description: "Call an action when your Snap is installed or updated."
+    },
+    {
+      icon: require("./assets/features/getfile.png").default,
+      href: "how-to/handle-static-files",
+      title: "Static File API",
+      description: "Lazy-load static files like Wasm modules or ZK keys"
```
Comment (Montoya) [inline snaps/index.mdx:91]: "Wasm modules or ZK circuits" not "keys"

File: snaps/how-to/handle-static-files.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+description: Include and retrieve static files in the Snap bundle.
+sidebar_position: 11
+---
+
+# Handle static files
+
+The Snaps platform provides a static file API that is useful to load Wasm modules, ZK keys, or anything else that needs to be lazily loaded.
```
Comment (Montoya) [inline snaps/how-to/handle-static-files.md:8]: "ZK circuits,"

**Review by Montoya — CHANGES_REQUESTED** [review]
Wording

**Comment by Montoya** [comment]
> Preview published: [zs/snap-getfile](https://docs.metamask.io/zs/snap-getfile/)

I don't know why the icon does not show up in this build

**Comment by ziad-saab** [comment]
> > Preview published: [zs/snap-getfile](https://docs.metamask.io/zs/snap-getfile/)
> 
> I don't know why the icon does not show up in this build

It definitely shows up for me. It's a base64 URL.

**Review by alexandratran — APPROVED** [review]
Moved this into the new "features" category and made some edits to related content. LGTM.

**Pull request merged**